static/app/views/replays/list/replayOnboardingPanel.tsx

```diff
@@ -298,7 +299,20 @@ export function SetupReplaysCTA({
         </Button>
       </ButtonList>
       <StyledWidgetContainer>
-        <StyledHeaderContainer>{t('FAQ')}</StyledHeaderContainer>
+        <StyledHeaderContainer>
+          {t('FAQ')}
+          {
+            <QuestionTooltip
+              size="xs"
+              isHoverable
+              title={tct('See a full list of FAQs [link:here].', {
```

Comment:
how about: (not sure this is the syntax):

```suggestion
              title={tct('See a [link:full list of FAQs].', {
```